fhir-persistence-jdbc/src/main/resources/fhirdb/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
File should be moved to an alternative folder.  it's not intended for production.

Reply:
Moved in next commit